### DISCUSSION TITLE
WIndows 11 x64

### DISCUSSION THREAD

**Discussion by 0xFantom** [vlang/v#14021] [Installation Issues]
I am trying to install v for windows 11 and it always gives me this error
```
 error: int cannot be compared with byte
 1951 |                                 break
 1952 |                             }
 1953 |                             if inner_c == wanted_c {
      |                                        ~~

Backend compiler error
```
The same error occurs if I try to start .\make.bat. I have tried installing for the website and from scoop and both have the same error

**Comment by JalonSolov** [comment]
Are you installing from a zip file, or cloning the repo?

Cloning the repo is the recommended method.  No idea who set up the "scoop" install, but there are _no_ official V installs, yet, other than cloning or getting the zip from github, and that is out-of-date almost as soon as it is created.

**Reply by 0xFantom** [reply]
I installed it from https://vlang.io/ (which is a zip file)

**Comment by ylluminate** [comment]
Did you review the following?: https://github.com/vlang/v/discussions/13768
